### PR TITLE
Add scram-engine and express-web-components

### PR DESCRIPTION
**Pull request by lastmjs**
I've been experimenting with server-side web components, allowing me to write server apps more declaratively with custom elements, and then running them on the server with a headless Electron instance. Electron offers a Chromium and Node.js environment, so the components can be rendered with the APIs provided by Chrome, and then you can call virtually any Node.js code in the JavaScript for each component. I'm writing the components with the Polymer library, and I've got a set of core Express elements that I've been using to write a few Express applications with. I've got a couple basic apps running on production environments on AWS right now. This potentially opens up many doors for server-side applications.

Full disclosure, I don't have any automated tests for these repos, but I have been manually testing them, and they are being used in various of my own applications. Also, documentation is not complete for each component in the express-web-components repo, but I do have documentation.

https://github.com/scramjs/scram-engine
https://github.com/scramjs/express-web-components

### CODE REVIEW

**Comment by lastmjs** [comment]
Also, maybe the section for scram-engine should be Tools -> Using Electron instead of Tools -> For Electron...

File: readme.md [hunk 1]
```diff
@@ -232,6 +233,7 @@ Made with Electron.
 - [electron-input-menu](https://github.com/parro-it/electron-input-menu) - Context menu for input elements.
 - [chrome-tabs](https://github.com/adamschwartz/chrome-tabs) - Chrome like tabs.
 - [titlebar](https://github.com/kapetan/titlebar) - Emulate the OS X window titlebar.
+- [express-web-components](https://github.com/scramjs/express-web-components) - Declaratively build Express.js applications with web components.
```
Comment (sindresorhus) [inline readme.md]: Out of scope for this list. The `Component` section is components you can use in the renderer to make apps, like widgets.

File: readme.md [hunk 2]
```diff
@@ -208,6 +208,7 @@ Made with Electron.
 - [spectron](https://github.com/electron/spectron) - Test Electron apps using ChromeDriver.
 - [babel-preset-electron](https://github.com/emorikawa/babel-preset-electron) - Babel preset that only compiles what's necessary for a particular Electron version.
 - [electron-is](https://github.com/delvedor/electron-is) - Useful 'is' utility functions.
+- [scram-engine](https://github.com/scramjs/scram-engine) - Write server apps declaratively with HTML and web components.
```
Comment (sindresorhus) [inline readme.md]: Should have been in `Using Electron`.

File: readme.md (hunk 1 above)
Comment (lastmjs) [inline readme.md]: express-web-components is a collection of web components that are used in the renderer to make apps, but they are server-side apps. They aren't UI, just functionality, but they are designed to work with a headless Electron instance (accessed with the scram-engine).

**Comment by sindresorhus** [comment]
Sounds like a cool idea, but I think it's too early to include in this list. It's just version 0.0.5, the idea is unproven, and there's almost no code. This list is also mainly intended for things that are useful for people making apps with Electron, so I'm trying to keep the `Using Electron` list short.

Btw, I would recommend showing off the idea to some Polymer team member and get feedback from them ;)

**Comment by lastmjs** [comment]
Thanks!

**Pull request closed without merging**